js/main.js

```diff
@@ -23,3 +39,47 @@ function updateImg(event) {
 
 form.elements['photo-url'].addEventListener('input', updateImg);
 form.addEventListener('submit', saveEntry);
+
+function loadEntry(entry) {
+  var columnOne = document.createElement('div');
+  columnOne.className = 'column-half';
+  var columnTwo = document.createElement('div');
+  columnTwo.className = 'column-half';
+  var img = document.createElement('img');
+  img.setAttribute('src', entry.photoUrl);
+  var notes = document.createElement('p');
+  notes.textContent = entry.notes;
+  var title = document.createElement('h2');
+  title.className = 'margin-top-1rem-wide-0';
+  title.textContent = entry.title;
+  var newEntry = document.createElement('div');
+  newEntry.className = 'row margin-bottom-2rem';
+  columnOne.append(img);
+  columnTwo.append(title, notes);
+  newEntry.append(columnOne, columnTwo);
+  return newEntry;
+}
+
+function appendEntries(event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    entries.appendChild(loadEntry(data.entries[i]));
+  }
+}
+
+function displayEntries() {
+  entryDisplay.className = 'container';
+  entryForm.className = 'container hidden';
+  data.view = 'entries';
+}
+
+function displayForm() {
+  entryDisplay.className = 'container hidden';
+  entryForm.className = 'container';
+  data.view = 'entry-form';
+}
```

Comment:
Instead of making an individual function definition for each view, I would make one function named something like `switchViews` that will receive an argument of a `string` containing the name of the view to show. Within that function, you should loop over all the possible views and check if they are the view name given as an argument. If it is that view, you should show it, and if it isn't you should hide it. You would also want to assign a value to `data.view` for whichever view is the correct view. Once you've done this, you'll have a reusable function you can use any time you need to move between views, no matter the context.